testem.js

```diff
@@ -25,7 +25,8 @@ module.exports = {
         '--disable-gpu',
         '--headless',
         '--remote-debugging-port=0',
-        '--window-size=1440,900'
+        '--window-size=1440,900',
+        '--no-sandbox'
```

Comment:
Can you swap this out to what the new addon blueprint does:

```js
browser_args: {
    Chrome: {
      mode: 'ci',
      args: [
        // --no-sandbox is needed when running Chrome inside a container
        process.env.TRAVIS ? '--no-sandbox' : null,

        '--disable-gpu',
        '--headless',
        '--remote-debugging-port=0',
        '--window-size=1440,900'
      ].filter(Boolean)
    }
  }
```

https://github.com/ember-cli/ember-new-output/blob/1bcb7a197e52ca01d1858de0346b789187a4b6c9/testem.js#L13-L21

Reply:
Done